### PR TITLE
Create es.json

### PR DESCRIPTION
**Pull request by KeruWolf**
Here's my proposed translation for the Turnip Prophet page for Spanish from issue  #177
This is my first time using Github so I hope I did everything correctly.
Regards!

### CODE REVIEW

File: locales/es.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  "first-time": {
+    "title": "Comprador(a) primerizo(a)",
+    "description": "¿Es esta la primera vez que has comprado nabos?<i>(Esta información afectará a tu patrón)</i>",
+    "yes": "Sí",
+    "no": "No"
+  },
+  "patterns": {
+    "title": "Patrón anterior",
+    "description": "¿Qué patrón describió el precio de los nabos la semana pasada?<i>(Esta información afectará a tu patrón)</i>",
+    "pattern": "Pattern",
```
Comment (mauriciabad) [inline locales/es.json:19]: Wouldn't this be translated as "Patrón"?

File: locales/es.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    "all": "All patterns",
```
Comment (mauriciabad) [inline locales/es.json:20]: Wouldn't this be translated as "Todos los patrones"?

File: locales/es.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    }
+  },
+  "textbox": {
+    "description": "Cuando introduzcas algunos precios, el Turnip Prophet empezará a hacer sus cálculos y te mostrará algunos posibles patrones para el precio de los nabos en tu isla.",
+    "development": "Esta aplicación está aún en desarrollo, ¡pero la seguiremos mejorando!",
+    "thanks": "Nada de esto habría sido posible sin el <a href=\"https://twitter.com/_Ninji/status/1244818665851289602?s=20\">trabajo de Ninji</a> para averiguar cómo Tendo y Nendo valoran los nabos.",
+    "support": "Para asistencia, comentarios y contribuciones, no dudes en pasarte por <a href=\"https://github.com/mikebryant/ac-nh-turnip-prices/issues\">Github</a>.",
+    "contributors-text": "¡Ah! ¡Y no nos olvidemos de todos los que han puesto su granito de arena hasta ahora!",
+    "contributors": "Contribuidores",
+    "language": "Language"
```
Comment (mauriciabad) [inline locales/es.json:78]: Wouldn't this be translated as "Idioma"?

**Comment by mauriciabad** [comment]
I think you also need to add the language into the languages variable in /js/translations.js line 16.

**Comment by hicor13** [comment]
Hi, I've read the file and all the translations are on point, I would only recommend the change I previously noted in line 18, that would be all.

**Comment by KeruWolf** [comment]
Hi @theRTC204 !
I translated the missed lines and added Spanish to de language selector.
Thanks @mauriciabad for pointing out the missing lines and all.

**Pull request merged**